### PR TITLE
Adding aqbanking-5.99.33_beta and gwenhywfar-4.99.16_beta

### PR DESCRIPTION
**Pull request by braegel**
Atoms are required for PSD2 support. Any financial transaction api in europe has to enforce this.

### CODE REVIEW

**Comment by scardracs** [comment]
Be sure to add both files on your manifest running ebuild *packagename* manifest and push Manifest again, this will fix you errors

File: net-libs/aqbanking/aqbanking-5.99.33_beta.ebuild [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=7
+
+DESCRIPTION="Generic Online Banking Interface"
+HOMEPAGE="https://www.aquamaniac.de/sites/aqbanking/index.php"
+MY_PV="5.99.33beta"
+S=${WORKDIR}/${PN}-${MY_PV}
+#S=${WORKDIR}/${PN}-${PV/-/_}
+SRC_URI="https://www.aquamaniac.de/rdm/attachments/download/145/${PN}-${MY_PV}.tar.gz"
```
Comment (scardracs) [inline net-libs/aqbanking/aqbanking-5.99.33_beta.ebuild:10]: Looks like link has been changed into https://www.aquamaniac.de/rdm/projects/aqbanking/files

File: net-libs/aqbanking/aqbanking-5.99.33_beta.ebuild [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2019 Gentoo Authors
```
Comment (scardracs) [inline net-libs/aqbanking/aqbanking-5.99.33_beta.ebuild:1]: 1999-2021

File: sys-libs/gwenhywfar/gwenhywfar-4.99.16_beta.ebuild [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+EAPI=7
+
+inherit qmake-utils
+
+DESCRIPTION="Multi-platform helper library for other libraries"
+HOMEPAGE="https://www.aquamaniac.de/sites/aqbanking/index.php"
+MY_PV="4.99.16beta"
+S=${WORKDIR}/${PN}-${MY_PV}
+SRC_URI="https://www.aquamaniac.de/rdm/attachments/download/140/${PN}-${MY_PV}.tar.gz"
```
Comment (scardracs) [inline sys-libs/gwenhywfar/gwenhywfar-4.99.16_beta.ebuild:12]: Same as before

**Review by scardracs — CHANGES_REQUESTED** [review]

**Comment by ajakk** [comment]
I suppose nobody's there :(

**Pull request closed without merging**